apps/desktop/src/views/SearchView/composables/useSearchWindowResize.ts

```
@@ -75,15 +85,47 @@ export function useSearchWindowResize(options: UseSearchWindowResizeOptions) {
     let unlistenWindowResize: (() => void) | null = null;
     let lastResizeConstraintsKey: string | null = null;
     let lastAllowHeightOverride: boolean | null = null;
     let lastSyncedDefaultSizeKey: string | null = null;
     let pendingDefaultSizeApplyAfterRestore = false;
+    let lastRequestedHeight: number | null = null;
+    let resizeTransactionInFlight = false;
+    let activeResizeDirection: 'grow' | 'shrink' | null = null;
+    let pendingProgrammaticTargetHeight: number | null = null;
+    let pendingObserverRemeasure = false;
+    let shrinkObserverReboundGuard:
+        | {
```

ESLint

[error] Delete `⏎········|`

(prettier/prettier)